# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/woff2/hotfix-cstdint.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- woff2-1.0.2/include/woff2/output.h.vanilla	2025-05-17 21:21:20.137891610 +0200
+++ woff2-1.0.2/include/woff2/output.h	2025-05-17 21:21:28.262404422 +0200
@@ -13,6 +13,7 @@
 #include <cstring>
 #include <memory>
 #include <string>
+#include <cstdint>
 
 namespace woff2 {
 
